packages/cactus-plugin-ledger-connector-fabric/src/main/json/openapi.json

```diff
@@ -1056,10 +1056,35 @@
               }
             }
           },
-          "skipDecode": {
-            "type": "boolean",
-            "description": "If true, encoded buffer will be returned. Otherwise, entire block object is returned.",
-            "default": false,
+          "type": {
```

Comment:
@outSH `type` is a reserved keyword in OpenAPI, is there any chance that this could be called something slightly more specific so that it doesn't collide with language features in the generator?

Reply:
@petermetz I've fixed it in previous PR where this change is introduced - https://github.com/hyperledger/cacti/pull/3298#issuecomment-2167706296. Thanks, great catch :)